### ISSUE TITLE
load_model before training

### ISSUE THREAD

**Issue by huanmit** [souravs17031999/Retinal_blindness_detection_Pytorch#10]
Hello,

Your work was brilliant and I was trying to follow your work to improve my accuracy. However, I noticed that you loaded a model before training. I want to know what is that model, the one you trained 100 epochs or something? Could you please explain this a little bit or could you upload the pretrained weight file? Thank you for your time!

**Comment by souravs17031999** [comment]
@florrietan Yes, I trained the model from scratch and saved it so that I could resume the training later on or use it while performing inference. You can use the same pretrained model.

Please check #6 

thanks
